src/js/assets/scss/colors.scss

```diff
@@ -8,6 +8,14 @@
 	--color-polls-background-maybe: #fcf7e1;
 }
 
+body#body-public,
```

Comment:
@dartcafe This doesn't just match in dark mode, but always.

Reply:
Ah damned. I did not test it in public polls.